Vic2ToHoI4/Data_Files/configurables/cultureGroupToGraphics.txt

```diff
@@ -1,27 +1,23 @@
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
 # Those with "# Revise" after it refers to generic portraits that should get a replacement
-# Currently 132 needed
+# Currently 280-ish needed
 # I support the idea of giving an historical figure a wrong name rather than using
 # John Jackson Adams, 1st Baron thereof
 
 germanic = {
-	army_portraits = {
```

Comment:
Instead of dropping these, equivalents should be added for the other culture groups. Though that should probably be in a new PR once this is merged.

Reply:
It was added in #880, which was the coding part of fixing #586 and #817

Reply:
I put them back. Does this mean we can add army and navy portraits to any culture group?